books/projects/quadratic-reciprocity/support/euclid.lisp

```diff
@@ -6,8 +6,8 @@
 
 ;;   (2) If p is a prime divisor of a*b, then p divides either a or b.
 
-(include-book "rtl/rel11/lib/basic" :dir :system) ;; Properties of fl and mod
-(include-book "rtl/rel11/lib/util" :dir :system)  ;; Utility macros
+(include-book "rtl/rel11/support/basic" :dir :system) ;; Properties of fl and mod
+(include-book "rtl/rel11/support/util" :dir :system)  ;; Utility macros
```

Comment:
I can imagine that @russinoff prefers to import `lib`, rather than the `support` books.  How come the change?